src/OrderTypes.php

```diff
@@ -2,31 +2,18 @@
 
 namespace num8er\TranzWarePaymentGateway;
 
+use Jawira\CaseConverter\Convert;
+
 class OrderTypes
 {
     const PURCHASE = 'Purchase';
     const PRE_AUTH = 'PreAuth';
 
     public static function sanitizeValue($value)
```

Comment:
In test testFromStringOneWordCaseFirst with $value = "Purchase", i got stack overflow iterations error.This pull request is resolve.All tests include latest, has worked well, i checked.